types/block.go

```diff
@@ -1637,7 +1637,8 @@ func (data *EvidenceData) splitIntoShares() NamespacedShares {
 		if err != nil {
 			panic(fmt.Sprintf("evidence included in evidence pool that can not be encoded %#v, err: %v", ev, err))
 		}
-		rawDatas = append(rawDatas, rawData)
+		delimetedRawData := append(append(make([]byte, 0, len(rawData)+1), byte(0)), rawData...)
```

Comment:
Can we introduce a constant or at least a local variable for the `1` used above. Mostly to document what it stands for.